cumulusci/salesforce_api/soap_envelopes.py

```diff
@@ -123,7 +123,7 @@
   </soap:Header>
   <soap:Body>
     <checkStatus xmlns="http://soap.sforce.com/2006/04/metadata">
-      <asyncProcessId>%(process_id)s</asyncProcessId>
+      <asyncProcessId>{process_id}</asyncProcessId>
     </checkStatus>
```

Comment:
Not strictly related to this PR, but while reviewing it I noticed that `checkStatus` is documented as having been removed in MD API version 31 and higher. So now I'm wondering why continuing to use it hasn't been a problem.

Reply:
We do override the status check envelope on the ApiDeploy subclass: https://github.com/SalesforceFoundation/CumulusCI/blob/b42bd95fd9fbf1fab0a8e7e6af7832af6f0513ce/cumulusci/salesforce_api/metadata.py#L341

My hunch is that we might be best to change them all to the new status check envelopes for each type.  Generally, all we do is deploy, retrieve, and list metadata.  Looks like retrieves are using the generic checkStatus still